includes/features/SU-SWS/permissions_tests/create_delete_affiliate_organization/create_delete_affiliate_organization.feature

```diff
@@ -0,0 +1,99 @@
+Feature: Ensure Site Owners can create and delete affiliates
+  In order to ensure that users with permissions can create, edit and delete affiliates
+  As a user with site permissions
+  I want to be able create, edit and delete affiliates
+
+  @api @dev @destructive @javascript
+  Scenario: Create, edit, then delete affiliate as site owner
```

Comment:
Do we need 4 scenarios? What about just two? one for site owner and one for editor? If we need all four, let's expand the description in the scenario as to "why" not just "what."